src/cascadia/TerminalControl/TSFInputControl.h

```diff
@@ -0,0 +1,101 @@
+#pragma once
+#include "TSFInputControl.g.h"
+#include "CursorPositionEventArgs.g.h"
+#include "FontInfoEventArgs.g.h"
+#include "cppwinrt_utils.h"
+
+namespace winrt::Microsoft::Terminal::TerminalControl::implementation
+{
+    struct CursorPositionEventArgs :
+        public CursorPositionEventArgsT<CursorPositionEventArgs>
+    {
+    public:
+        CursorPositionEventArgs(){};
+
+        Windows::Foundation::Point CurrentPosition()

```

Comment:
you can probably use `GETSET_PROPERTY` and clean up a bunch of this manual stuff

Reply:
Totally agree, will be in next PR push.